src/main/scala/org/wikimedia/analytics/hdfstools/HdfsRsyncCLI.scala

```diff
@@ -76,44 +111,95 @@ object HdfsRsyncCLI {
 
         opt[Unit]('v', "verbose")
             .optional()
-            .action((_, c) => c.copy(logLevel = Level.DEBUG))
-            .text("Add verbosity to logging (DEBUG)")
+            .action((_, c) => c.copy(applicationLogLevel = Level.DEBUG))
+            .text("Add verbosity to application logging (DEBUG)")
 
         opt[Unit]('q', "quiet")
             .optional()
-            .action((_, c) => c.copy(logLevel = Level.WARN))
-            .text("Remove verbosity from logging (WARN)")
+            .action((_, c) => c.copy(applicationLogLevel = Level.WARN))
+            .text("Remove verbosity from application logging (WARN)")
+
+        opt[Unit]("all-logs-info")
+            .optional()
+            .action((_, c) => c.copy(rootLogLevel = Level.INFO))
+            .text("Set root logger level to INFO (default: ERROR)")
+
+        opt[Unit]("all-logs-debug")
+            .optional()
+            .action((_, c) => c.copy(rootLogLevel = Level.DEBUG))
+            .text("Set root logger level to DEBUG (default: ERROR)")
+
+        opt[String]("log-file")
+            .optional()
+            .action((x, c) => c.copy(logFile = Some(x)))
+            .text("File to write logs instead of sending them to stdout")
```

Comment:
Do we need this?  Can we just use redirection from the CLI, or log4j properties files?

Reply:
We can redirect from CLI for sure. This option was there as a facility. I can remove it if you prefer :)

Reply:
I think it might just cause confusion.  Let's remove.

Reply:
Done